funnel/views/schedule.py

```diff
@@ -176,19 +195,36 @@ class ProjectScheduleView(ProjectViewMixin, UrlForView, ModelView):
     def schedule(self):
         schedule_transition_form = ProjectScheduleTransitionForm(obj=self.obj)
         project_save_form = SavedProjectForm()
+        scheduled_sessions_list = session_list_data(
+            self.obj.scheduled_sessions, with_modal_url='view_popup'
+        )
         return {
             'project': self.obj,
-            'from_date': date_js(self.obj.schedule_start_at),
-            'to_date': date_js(self.obj.schedule_end_at),
-            'sessions': session_list_data(
-                self.obj.scheduled_sessions, with_modal_url='view_popup'
+            'from_date': (
+                localize_timezone(
+                    self.obj.schedule_start_at, tz=self.obj.timezone
+                ).isoformat()
+                if self.obj.schedule_start_at
+                else None
+            ),
+            'to_date': (
+                localize_timezone(
+                    self.obj.schedule_end_at, tz=self.obj.timezone
+                ).isoformat()
+                if self.obj.schedule_start_at
+                else None
             ),
-            'timezone': self.obj.timezone.utcoffset(datetime.now()).total_seconds(),
+            'sessions': scheduled_sessions_list,
+            'timezone_offset': self.obj.timezone.utcoffset(datetime.now()).total_seconds(),
```

Comment:
Use this expression. It will pick the offset for the event start date, if present:
`(self.obj.schedule_start_at.astimezone(self.obj.timezone) if self.obj.schedule_start_at else datetime.now(self.obj.timezone)).utcoffset().total_seconds()`

Reply:
Frontend is using timezone to calculate the date & time so removing the timezone offset